client/app/routes/landuse-form.js

```diff
@@ -4,10 +4,19 @@ import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-rout
 export default class LanduseFormRoute extends Route.extend(AuthenticatedRouteMixin) {
   authenticationRoute = '/';
 
+  packageSideloads = [
+    'landuse-form.bbls',
+    'landuse-form.applicants',
+    'project',
+    'landuse-form.related-actions',
+    'landuse-form.landuse-actions',
+    'landuse-form.sitedatah-forms',
+  ];
+
   async model(params) {
     const landuseFormPackage = await this.store.findRecord('package', params.id, {
       reload: true,
-      include: 'landuse-form.bbls,landuse-form.applicants,project,landuse-form.related-actions,landuse-form.landuse-actions',
+      include: this.packageSideloads.join(),
```

Comment:
The line was getting too long and unwieldy, so wanted to break it out vertically.

Reply:
cool! maybe just in-line the array.